Python/compile.c

```diff
@@ -5652,8 +5663,12 @@ compiler_comprehension(struct compiler *c, expr_ty e, int type,
         }
     }
     else {
+        /* Receive outermost iter as an implicit argument */
+        _PyCompile_CodeUnitMetadata umd = {
+            .u_argcount = 1,
+        };
         if (compiler_enter_scope(c, name, COMPILER_SCOPE_COMPREHENSION,
-                                (void *)e, e->lineno, NULL, NULL) < 0) {
+                                (void *)e, e->lineno, NULL, &umd) < 0) {
```

Comment:
```suggestion
                                 (void *)e, e->lineno, NULL, &umd) < 0) {
```

I *think* the `(` of `(void *` should be aligned with the `c`. I may have incorrectly changed this one once.